_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -75372,8 +75372,10 @@
 /obj/machinery/door/firedoor,
 /obj/machinery/power/apc{
 	areastring = "/area/hallway/secondary/entry";
+	dir = 8;
 	name = "Arrivals APC";
-	pixel_x = -26
+	pixel_x = -26;
+	pixel_y = null
```

Comment:
Remove pixel_y and the ; at the end of pixel_x. Do it manually in vscode or something, no mapper needed.